src/app/cases/viewer/case-viewer.component.ts

```diff
@@ -105,4 +105,8 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
       return acc.concat(tab.fields);
     }, []);
   }
+
+  isDraft(): boolean {
+    return this.caseDetails.case_id.length < 16;
```

Comment:
in the resolver we use a different strategy to check whether a case is a draft, i.e. cid.startsWith(CaseResolver.DRAFT)
Can we use one approach only? 
Also, I would suggest introducing a method 
`isDraft(cid)`
somewhere, e.g. on CaseDetails,
 to avoid having this logic duplicated in multiple points